RealmSwift/Optional.swift

```diff
@@ -43,7 +43,7 @@ extension Bool: RealmOptionalType {}
 
  To change the underlying value stored by a `RealmOptional` instance, mutate the instance's `value` property.
  */
-@available(*, deprecated, renamed: "RealmProperty", message: "RealmOptional has been deprecated, use RealmProperty<T?> instead.")
+@available(*, deprecated, renamed: "RealmProperty", message: "RealmOptional<T> has been deprecated, use RealmProperty<T?> instead.")
```

Comment:
`T?` seems wrong? If we go by other wrappers, it would be represented as `T` and can be optional or not optional.

Reply:
with RealmOptional T was an optional type. With the introduction of RealmProperty you are able to box optional and non optional properties

Reply:
Ah, I see. It's a catchall wrapper now. This notation does make sense

Reply:
Yeah, I'm trying to hint "you need to add a ? here" because it isn't quite just renaming the existing type.